webpack.webext.babel.js

```diff
@@ -44,4 +47,12 @@ config.plugin('copy').tap(([patterns]) => [[
   },
 ]]);
 
+config.plugin('zip')
+  .use(ZipWebpackPlugin, [
+    {
+      path: path.join(__dirname, 'dist'),
```

Comment:
Ah, this is what the `dist` helper is for 😉 I'll create a quick, tiny follow-up PR

Reply:
Argh, and maybe we want to run zip each time when running in watch mode, so a separate task might be better.